lua/lazyvim/plugins/extras/lang/jsonnet.lua

```diff
@@ -0,0 +1,28 @@
+return {
+  { "google/vim-jsonnet" },
+  {
+    "neovim/nvim-lspconfig",
+    opts = {
+      servers = {
+        -- Ensure mason installs the server
+        jsonnet_ls = {},
+      },
+    },
+  },
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "jsonnet" })
+      end
+    end,
+  },
+  {
+    "williamboman/mason.nvim",
+    opts = function(_, opts)
+      vim.list_extend(opts.ensure_installed, {
+        "jsonnet-language-server",
+      })
+    end,
+  },
```

Comment:
```suggestion
```
This is not needed, as LazyVim automatically installs the servers that are defined in `nvim-lspconfig.opts.servers`. 
For more info, how this is set up, see [here](https://github.com/LazyVim/LazyVim/blob/97480dc5d2dbb717b45a351e0b04835f138a9094/lua/lazyvim/plugins/lsp/init.lua#L201-L216).

Reply:
Oh, I wasn't aware, Thanks